docs/Explorer.md

```diff
@@ -8,6 +8,16 @@ The Kafka explorer shows configured clusters with their topics, brokers, consume
 
 ![Screenshot-3](assets/screen-3.png)
 
+## Cluster Item
+
+The explorer shows clusters, click on right button on cluster provides several menu items.

```

Comment:
In the Kafka explorer, right-click on a cluster to access several options.

Reply:
fixed